stdlib/log/logger.lua

```diff
@@ -77,7 +91,11 @@ function Logger.new(mod_name, log_name, debug_mode, options)
             end
         else
             if _G.script then --buffer when a save is loaded but _G.game isn't available
-                table.insert(Logger.buffer, format("00:00:00: %s\n", msg))
+                if Logger.options.log_ticks then
```

Comment:
Another Irk of mine when using the log, any pre game buffer was indented differently